### PR TITLE
Update contracts to v0.6.0-beta1

### LINKED ISSUE #248: Update tgrade to v0.6.0-beta1 contracts
This should have all bugfixes and functional changes

### PR DESCRIPTION
**Pull request by ethanfrey**
Closes #248 

I just adjusted existing queries/messages. There are many variants that are not covered. It would be good to do a thorough update of all structs, but maybe wait til 0.6.0 final is cut and port them all over then

TODO:
- proper pagination on the active validators query (if we use this somewhere)

### CODE REVIEW

File: x/poe/contract/download_releases.sh [hunk 1]
```diff
@@ -8,11 +8,21 @@ if [ $# -ne 1 ]; then
 fi
 
 tag="$1"
+
 # load token from OS keychain when not set via ENV
 GITHUB_API_TOKEN=${GITHUB_API_TOKEN:-"$(security find-generic-password -a "$USER" -s "github_api_key" -w)"}
 
 rm -f version.txt
-for contract in tg4_engagement tgrade_valset tg4_mixer tg4_stake tgrade_trusted_circle tgrade_gov_reflect tgrade_oc_proposals tgrade_community_pool tgrade_validator_voting; do
+for contract in tg4_engagement tgrade_valset tg4_mixer tg4_stake tgrade_gov_reflect tgrade_community_pool tgrade_validator_voting; do
```
Comment (maurolacy) [inline x/poe/contract/download_releases.sh:16]: `tgrade_gov_reflect` can be removed from the list.

File: x/poe/contract/tgrade_valset.go [hunk 2]
```diff
@@ -75,7 +75,13 @@ type TG4ValsetExecute struct {
 	// only if the jail period passed.
 	Unjail *UnjailMsg `json:"unjail,omitempty"`
 	// UpdateAdmin set a new admin address
-	UpdateAdmin *TG4UpdateAdminMsg `json:"update_admin,omitempty"`
+	UpdateAdmin  *TG4UpdateAdminMsg `json:"update_admin,omitempty"`
```
Comment (maurolacy) [inline x/poe/contract/tgrade_valset.go:78]: Nit: you are enforcing the indentation here, but not above.

File: x/poe/contract/tgrade_valset.go [hunk 3]
```diff
@@ -85,17 +91,8 @@ type JailMsg struct {
...
-
-func (j JailingDuration) MarshalJSON() ([]byte, error) {
-	if j.Forever {
-		return json.Marshal("forever")
-	}
-	return json.Marshal(struct {
-		Duration uint64 `json:"duration,omitempty"`
-	}{Duration: j.Duration})
+	Duration uint64    `json:"duration,omitempty"`
+	Forever  *struct{} `json:"forever,omitempty"`
```
Comment (maurolacy) [inline x/poe/contract/tgrade_valset.go:95]: Nice.

File: x/poe/contract/tgrade_valset.go [hunk 4]
```diff
@@ -194,10 +191,11 @@ type ValsetEpochResponse struct {
...
 type OperatorResponse struct {
-	Operator    string            `json:"operator"`
-	Pubkey      ValidatorPubkey   `json:"pubkey"`
-	Metadata    ValidatorMetadata `json:"metadata"`
-	JailedUntil *JailingPeriod    `json:"jailed_until,omitempty"`
+	Operator        string            `json:"operator"`
+	Pubkey          ValidatorPubkey   `json:"pubkey"`
+	Metadata        ValidatorMetadata `json:"metadata"`
+	JailedUntil     *JailingPeriod    `json:"jailed_until,omitempty"`
+	ActiveValidator bool              `json:"active_validator"`
```
Comment (maurolacy) [inline x/poe/contract/tgrade_valset.go:198]: :-)

**Review by maurolacy — APPROVED** [review]
LGTM.

**Comment by maurolacy** [comment]
> TODO:
> 
> * proper pagination on the active validators query (if we use this somewhere)

I can implement this in a follow-up.

File: x/poe/contract/download_releases.sh (hunk 1 above)
Comment (alpe) [reply x/poe/contract/download_releases.sh:16]: `tgrade_gov_reflect` is still used in system tests

File: x/poe/contract/tgrade_valset.go (hunk 2 above)
Comment (ethanfrey) [reply x/poe/contract/tgrade_valset.go:78]: `go fmt` does that (and yeah, I find it annoying that it resets when there is a comment)

**Pull request merged**